decidim-meetings/config/locales/en.yml

```diff
@@ -64,6 +64,10 @@ en:
           unfold: Unfold
         index:
           view_meeting: View meeting
+        meetings:
+          no_meetings_warning: At the moment, this process doesn't have any meeting scheduled.
```

Comment:
This seems a weird translation from Catalan, what about:

"No meetings match your search criteria or no there isn't any meeting scheduled".